functions-common

```diff
@@ -1028,6 +1058,8 @@ function _get_package_dir {
         pkg_dir=$base_dir/rpms
     elif is_suse; then
         pkg_dir=$base_dir/rpms-suse
+    elif is_clear;then
+        pkg_dir=$base_dir/rpms
```

Comment:
This needs to be a new directory because names are not the same... use suse as the example here

Reply:
right fix WIP :)
thanks for review